src/javascripts/lib/i18n.js

```diff
@@ -32,7 +32,8 @@ class I18n {
 
   tryRequire (locale) {
     try {
-      return require(`../../translations/${locale}.json`)
+      // This path matches what the test is mocking
```

Comment:
Not really a relevant comment anymore, we're mocking the entire tryRequire method in our test